.npmrc

```diff
@@ -1 +0,0 @@
-//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
Hmm, not sure why I checked in this file anymore, thanks for removing it

Reply:
This was fine, but `action/setup-node` does some magic that allow to do it without having this kind of file laying around